moralization/data_manager.py

```diff
@@ -48,15 +49,15 @@ def occurence_analysis(self, _type="table", cat_filter=None, file_filter=None):
                 self.occurence_df, _type=_type, _filter=cat_filter
             )
 
-    def interactive_analysis(self):
+    def interactive_correlation_analysis(self):
```

Comment:
should this still be run in the `data_manager_test` notebook? If so, function name needs changing.